cmd/monitor/monitor_edit.go

```diff
@@ -80,6 +86,13 @@ func RunMonitorEditCommand(args *EditArgs) di.CmdWithApiFn {
 			return nil
 		}
 
+		if args.Unlock {
+			_, resp, err := api.NodeApi.Unlock(cli.Context, MonitorNodeType, id).Execute()
+			if err != nil {
+				return common.NewResponseError(err, resp)
```

Comment:
What does the call return if the node was not locked to begin with?

Reply:
It's idempotent - returns the same as if it were just unlocked.